affiliates/mexico.swagger.json

```diff
@@ -438,6 +438,11 @@
             "description": "Required when `credit_score` = DONT_KNOW",
             "example": true
           },
+          "car" : {
```

Comment:
should be documented when it is required... it is not always required

Reply:
ah, shit, I am stupid and cannot read..

Reply:
added description